code/game/objects/items/weapons/storage/garment.dm

```diff
@@ -195,12 +195,16 @@
 	new /obj/item/clothing/under/rank/cargo/quartermaster(src)
 	new /obj/item/clothing/under/rank/cargo/quartermaster/skirt(src)
 	new /obj/item/clothing/shoes/brown(src)
-	new /obj/item/clothing/mask/gas(src)
+	new /obj/item/clothing/suit/hooded/wintercoat/cargo(src)
+	new /obj/item/clothing/suit/jacket/cargobomber(src)
+	new /obj/item/clothing/head/hardhat/orange(src)
+	new /obj/item/clothing/suit/storage/hazardvest(src)
+	new /obj/item/clothing/mask/gas/explorer(src)
+	new /obj/item/clothing/suit/hooded/explorer(src)
```

Comment:
Going to keep the explorer suit?

Loader modsuits allows people to survive in lavaland? I dont know much about modsuits.

Reply:
Nah the loader is definitely more of a 'delivering crates' than Lavaland thing. I threw the suit in for the occasional lowpop scenario of the QM having to do a small bit of mining, or occasional rescue.